### PR TITLE
Info on loading duplicate data

### PR DESCRIPTION
**Pull request by tanevitt**
Added info on handling the case of trying to load duplicate data.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,11 +77,13 @@ It is also necessary to include an [.env file](#env-file) within the working dir
 
 Finally, the data must be exported from the cycler in a [specific format](#data-export-requirements)
 
-For a practical implementation of `BattETL`, please refer to the example notebook `battetl_demo.ipynb` located in the `examples` directory. This notebook demonstrates how to implement `BattETL` and can serve as a useful reference for your own project.
+Note that repeated calls to load test data that already exists in the database (as determined by unix timestamp) will not overwrite any existing data. Only test data after the lastest existing unix timestamp will be loaded. In the case of loading cycle statistics, any duplicate cycles that already exist in the database will be overwritten.
```
Comment (chintanp) [inline README.md:80]: @tanevitt can you reformat this a bit? One suggestion is to make it a quote: https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-text 

This is an important design decision, that should be highlighted.

**Review by chintanp — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tanevitt) [reply README.md]: Not sure how to highlight this further. I put it in quotes as suggested. Feel free to edit the PR for any changes you think would help draw attention to this.

**Review by chintanp — APPROVED** [review]
LGTM!

**Review by SyXuan — APPROVED** [review]
LGTM!

**Pull request merged**